maven-repository-metadata/src/site/apt/index.apt

```diff
@@ -27,26 +27,32 @@ Maven Repository Metadata Model
 
  This is strictly the model for Maven Repository Metadata, so really just plain objects.
 
- Maven Repository Metadata is available in directories representing:
-
- [[1]] an un-versioned artifact: it gives informations about available versions of the artifact,
+ The metadata file name is:
 
- [[2]] a snapshot artifact: it gives precise information on the snapshot,
+ * <<<maven-metadata.xml>>> in a remote repository,
 
- [[3]] a group containing Maven plugins artifacts: it gives informations on plugins available in this group.
+ * <<<maven-metadata-\<repo-id>.xml>>> in a local repository, for metadata from a repository with <<<repo-id>>> identifier.
 
  []
 
- The metadata file name is:
+ Depending on what the directory represents ("groupId", "groupId/artifactId" or "groupId/artifactId/version"),
+ the Maven Repository Metadata file contains 3 different sets of metadata:
 
- * <<<maven-metadata.xml>>> in a remote repository,
+ [[1]] in a "groupId" directory: a "groupId" directory may contain Maven plugins artifacts, which are described in metadata's <<<plugins>>> element,
 
- * <<<maven-metadata-\<repo-id>.xml>>> in a local repository, for metadata from a repository with <<<repo-id>>> identifier.
+ [[2]] in a "groupId/artifactId" directory: metadata describes <<<groupId>>>, <<<artifactId>>> and <<<versioning>>> element that
+       gives data about available versions (<<<latest>>>, <<<release>>>, <<<versions>>> list and <<<lastUpdated>>>),
+
+ [[3]] in a "groupId/artifactId/version" snapshot artifact directory: metadata describes <<<groupId>>>, <<<artifactId>>>, <<<version>>> (base version, ie ending in <<<-SNAPSHOT>>>) and
+       <<<versioning>>> element that gives data about snaphot (<<<snapshot>>>, <<<lastUpdated>>> and <<<snapshotVersions>>> list). Notice that a
+       release artifact directory is not expected to provide metadata.
 
  []
 
  The following are generated from this model:
 
-   * {{{./apidocs/index.html}Java sources}} with Reader and Writers for the Xpp3 XML parser, to read and write <<<maven-metadata(-*).xml>>> files
+   * {{{./apidocs/index.html}Java sources}} with Reader and Writers for the Xpp3 XML parser, to read and write <<<maven-metadata(-*).xml>>> files,
 
-   * A {{{./repository-metadata.html}Descriptor Reference}}
+   * a {{{./repository-metadata.html}Descriptor Reference}}.
+ 
+ Notice: data about plugins in a directory representing a groupId is deprecated and will be removed in a future Maven version (perhaps 4)
```

Comment:
Is there a JIRA issue for this?

Reply:
it's a consequence of removing maven-compat: I don't know the MNG for it

Reply:
Maybe you should just mention that.

Reply:
or more precisely: removing maven-compat will stop updating the value
removing the field is IMHO more clear than having half updated values

Reply:
I can remove ht "maybe 4" part that is both too vague and too precise

Reply:
> it's a consequence of removing maven-compat: I don't know the MNG for it

https://issues.apache.org/jira/browse/MNG-7266 ?